monitoring/uss_qualifier/rid/aircraft_state_replayer.py

```diff
@@ -111,21 +100,5 @@ def submit_test(self, payload: CreateTestParameters, test_id: str, setup: report
 
         if response.status_code == 200:
             print("New test with ID %s created" % test_id)
-
-        elif response.status_code == 409:
-            raise RuntimeError("Test with ID %s already exists" % test_id)
-
-        elif response.status_code == 404:
-            raise RuntimeError("Test with ID %s not submitted, the requested endpoint was not found on the server" % test_id)
-
-        elif response.status_code == 401:
-            raise RuntimeError("Test with ID %s not submitted, the access token was not provided in the Authorization header, or the token could not be decoded, or token was invalid" % test_id)
-
-        elif response.status_code == 403:
-            raise RuntimeError("Test with ID %s not submitted, the access token was decoded successfully but did not include the appropriate scope" % test_id)
-
-        elif response.status_code == 413:
-            raise RuntimeError("Test with ID %s not submitted, the injection payload was too large" % test_id)
-
         else:
-            raise RuntimeError("Test with ID %(test_id)s not submitted, the server returned the following HTTP error code: %(status_code)d %(msg)s" % {'test_id': test_id, 'status_code': response.status_code, 'msg': response.content})
+            raise RuntimeError('Error {} submitting test ID {}: {}'.format(response.status_code, test_id, response.content.decode('utf-8')))
```

Comment:
It may be useful to log the target uss.

Reply:
This harness doesn't actually know the USS exactly at this point, but I've added the base URL to the message since that will probably give a good indication.